packages/app-cli/tests/MdToHtml.ts

```diff
@@ -57,10 +56,8 @@ describe('MdToHtml', function() {
 			const result = await mdToHtml.render(markdown, null, mdToHtmlOptions);
 			let actualHtml = result.html;
 
-			if (os.EOL === '\r\n') {
-				expectedHtml = expectedHtml.replace(/\r\n/g, '\n');
-				actualHtml = actualHtml.replace(/\r\n/g, '\n');
-			}
+			expectedHtml = expectedHtml.replace(/\r?\n/g, '\n');
```

Comment:
Some unrelated tests were failing for me because of this. The reason is line endings depend on git config and do not have to conform to OS convention. Which was the case for me -- expected html had extra `\r` at the end of each line.

Reply:
Yes I remember someone added this because it wasn't working for them on Windows. It's a bit annoying we have to deal with changing line endings in tests actually, we should probably enforce git line ending at the repo level.